### ISSUE TITLE
The semantic segmentation masks don't always show up after I submitted the result

### ISSUE THREAD

**Issue by chktrp** [heartexlabs/label-studio#668]
**Describe the bug**
The semantic segmentation masks don't always show up after I submitted the result.

**To Reproduce**
1. Create a Semantic Segmentation with Masks Project
2. Import an image
3. Draw a brushstroke
4. Submit
5. After the image is reloaded, the brushstroke doesn't show up

**Screenshots**
![image](https://user-images.githubusercontent.com/23154789/111572010-6611e080-87da-11eb-9be8-618632ffbb7c.png)

Task source
```
{
  "id": 7,
  "data": {
    "image": "/data/upload/plane.png"
  },
  "annotations": [
    {
      "id": 8,
      "created_username": " (test@test.com, 1)",
      "created_ago": "16 minutes",
      "completed_by": {
        "id": 1,
        "first_name": "",
        "last_name": "",
        "username": "test",
        "email": "test@test.com",
        "last_activity": "2021-03-18T04:01:19.982174Z",
        "avatar": null,
        "initials": "te",
        "phone": ""
      },
      "ground_truth": false,
      "result": [],
      "was_cancelled": false,
      "created_at": "2021-03-18T03:44:40.695788Z",
      "updated_at": "2021-03-18T03:52:42.245819Z",
      "lead_time": 1178.574,
      "task": 7
    }
  ],
  "predictions": []
}
```

**Environment (please complete the following information):**
 - OS: Ubuntu
 - Label Studio Version 1.0.0

**Comment by EldorIbragimov** [comment]
Yes, It is not saving annotations. Also, I tried  'brush labels to PNG' option. But it does not work.

**Comment by makseq** [comment]
@EldorIbragimov The issue with export was fixed in LS 1.0.0.post2, please do `pip install -U label-studio`. 
The disappearing brushes are fixed in the same .post2 I hope :slightly_smiling_face:

**Comment by chktrp** [comment]
It worked! Thank you very much!
